lib/src/component_declaration/flux_component.dart

```diff
@@ -82,6 +83,7 @@ abstract class FluxUiStatefulComponent<TProps extends FluxUiProps, TState extend
 ///
 /// Private so it will only get used in this file, since having lifecycle methods in a mixin is risky.
 abstract class _FluxComponentMixin<TProps extends FluxUiProps> implements BatchedRedraws {
+  final Logger _logger = new Logger('_FluxComponentMixin');
```

Comment:
Mm, good idea; we don't want this potentially leaking for every FluxComponent instance.

@tomconnell-wf What about making this instance static / top-level instead?